apps/meteor/app/models/server/models/Users.js

```diff
@@ -202,10 +202,10 @@ export class Users extends Base {
 		return this.find(query);
 	}
 
-	getNextAgent(ignoreAgentId, extraQuery) {
+	async getNextAgent(ignoreAgentId, extraQuery) {
```

Comment:
can this be changed to `async`? I mean, not sure who is calling this so idk if they're "ready" to receive a `Promise` as return

Reply:
Hey, yups, those places were already expecting a promise (since other routing methods returned promises) so we're good there.